developer-experience/readme.md

```diff
@@ -74,6 +74,13 @@ The team members of the team can also help hold each other accountable for provi
   - Introduction of new tech whose testability is limited to manual steps in a deployed environment.
   - Addition of new repository
 
+### New Team Members - Identify Iterative Improvements
+
+New team members are uniquely positioned to identify instances of undocumented [Collective Wisdom](https://en.wikipedia.org/wiki/Collective_wisdom). The following outlines responsibilities of new team members as it relates to devex:
+
+- If you come across missing, incomplete or incorrect documentation while onboarding, you should open a PR to fix it.
+- Contribute to the developer onboarding plan if you find there to be missing steps or information. If no plan exists, create one.

```

Comment:
I dont necessarily agree with fixing these right on the spot. Taking this kind of action takes power away from the Product Owner in prioritizing **what** the dev team should be delivering. It could also put existing sprint commitments at risk. Lastly, we wouldnt want to open a PR without a linked work item. We would need something to track the work. Therefore, I would usually recommend that the person that discovered the issue start by recording the issue in defect or story so the product owner can decide when this work should be completed. 

- Should the onboarding documentation be found to be missing, incomplete, or incorrect, record the issues within a new defect(s) and assign to the product owner to be triaged.
- Should no onboarding documentation exist, note the steps taken. Then submit a new user story to the product owner to formally document the onboarding process using the notes taken.

Reply:
Great points. This is a good way to shift this decision to the product owner, who can assess the priority with respect to the other work the team is taking on. Thanks!